lib/node_modules/@stdlib/stats/base/dsnanmean/manifest.json

```diff
@@ -24,17 +27,83 @@
   ],
   "confs": [
     {
+      "task": "build",
+      "wasm": false,
       "src": [
-        "./src/dsnanmean.c"
+        "./src/main.c"
       ],
       "include": [
         "./include"
       ],
-      "libraries": [
-        "-lm"
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/blas/base/shared",
+        "@stdlib/strided/base/stride2offset",
+        "@stdlib/math/base/assert/is-positive-zero",
+        "@stdlib/math/base/assert/is-nan",
+        "@stdlib/stats/base/dsnanmeanpn",
+        "@stdlib/napi/export",
+        "@stdlib/napi/argv",
+        "@stdlib/napi/argv-int64",
+        "@stdlib/napi/argv-strided-float32array",
+        "@stdlib/napi/create-double"
+      ]
+    },
+    {
+      "task": "benchmark",
+      "wasm": false,
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/blas/base/shared",
+        "@stdlib/strided/base/stride2offset",
+        "@stdlib/math/base/assert/is-nan",
+        "@stdlib/math/base/assert/is-positive-zero",
+        "@stdlib/stats/base/dsnanmeanpn"
+      ]
+    },
+    {
+      "task": "examples",
+      "wasm": false,
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/blas/base/shared",
+        "@stdlib/strided/base/stride2offset",
+        "@stdlib/math/base/assert/is-nan",
+        "@stdlib/math/base/assert/is-positive-zero",
```

Comment:
TMK, neither of these dependencies are used. Please don't just blindly copy-paste things from other packages. Think about what you are doing and whether something is applicable to the specific logic/implementation at hand.